content/posts/irods-development-update-january-2025.md

```diff
@@ -0,0 +1,296 @@
+Title: iRODS Development Update: January 2025
+Date: 2025-01-29 12:00
+Author: Kory Draughn
+Slug: irods-development-update-january-2025
+Status: published
+
+
+Happy New Year!
+
+I hope you had a wonderful and safe holiday. People are still getting back into the swing of things, so this will be a short update.
+
+TRiRODS will be happening on February 12th. Justin James, of the iRODS Consortium, will be giving a presentation on the upcoming Metalnx 3.0.0 release. You can register and find more information about the event at [https://irods.org/trirods/](https://irods.org/trirods/). We hope to see you there.
+
+Efforts to release Storage Tiering capability plugin 4.3.3.1 are in progress. This upcoming release is all about improving stability and addressing bugs. To help with tracking down memory errors, we've added support for Address Sanitizer.
+
+Python iRODS client (PRC) 3.0.0 is now available! It was released on December 19, 2024 and closed 26 issues. The most important change of the release is the removal of Python 2 compatibility. With that comes improved handling of the **.irodsA** file and improved support for PAM authentication. To learn more, see the release notes at [https://github.com/irods/python-irodsclient/releases/tag/v3.0.0](https://github.com/irods/python-irodsclient/releases/tag/v3.0.0). A new release doesn't mean the work stops there though. We're now working to bring full compatibility with the iRODS 4.3 authentication framework, rsync functionality, and enhancements to I/O buffering.
```

Comment:
'rsync' is probably too specific... i'd say just 'sync'

Reply:
Updated.